### PR TITLE
git-describe: add French translation

### PR DESCRIPTION
**Pull request by antoine-amara**
add french translation for git-describe sub-command.

- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.fr/common/git-describe.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# git describe
+
+> Créer un nom unique et lisible pour un objet à partir d’une référence disponible.
+> Plus d'informations : <https://git-scm.com/docs/git-describe>.
+
+- Créer un nom unique pour le commit courant (le nom contient le tag le plus recent, le nombre de commits additionnel, and le hash abrégé du commit):
+
+`git describe`
+
+- Créer un nom avec un hash de commit de 4 lettres:
```
Comment (navarroaxel) [inline pages.fr/common/git-describe.md:10]:
```suggestion
- Créer un nom avec un hash de commit de 4 lettres :
```

File: pages.fr/common/git-describe.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Générer un nom avec le chemin complet du tag:
+
+`git describe --all`
+
+- Décrire un tag:
+
+`git describe {{v1.0.0}}`
+
+- Créer un nom pour le dernier commit d'une branche donnée:
```
Comment (navarroaxel) [inline pages.fr/common/git-describe.md:22]:
```suggestion
- Créer un nom pour le dernier commit d'une branche donnée :
```

File: pages.fr/common/git-describe.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Créer un nom avec un hash de commit de 4 lettres:
+
+`git describe --abbrev={{4}}`
+
+- Générer un nom avec le chemin complet du tag:
+
+`git describe --all`
+
+- Décrire un tag:
```
Comment (navarroaxel) [inline pages.fr/common/git-describe.md:18]:
```suggestion
- Décrire un tag :
```

File: pages.fr/common/git-describe.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Créer un nom unique pour le commit courant (le nom contient le tag le plus recent, le nombre de commits additionnel, and le hash abrégé du commit):
+
+`git describe`
+
+- Créer un nom avec un hash de commit de 4 lettres:
+
+`git describe --abbrev={{4}}`
+
+- Générer un nom avec le chemin complet du tag:
```
Comment (navarroaxel) [inline pages.fr/common/git-describe.md:14]:
```suggestion
- Générer un nom avec le chemin complet du tag :
```

File: pages.fr/common/git-describe.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# git describe
+
+> Créer un nom unique et lisible pour un objet à partir d’une référence disponible.
+> Plus d'informations : <https://git-scm.com/docs/git-describe>.
+
+- Créer un nom unique pour le commit courant (le nom contient le tag le plus recent, le nombre de commits additionnel, and le hash abrégé du commit):
```
Comment (navarroaxel) [inline pages.fr/common/git-describe.md:6]: Please use the space before the `:`.
```suggestion
- Créer un nom unique pour le commit courant (le nom contient le tag le plus recent, le nombre de commits additionnel, and le hash abrégé du commit) :
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @antoine-amara, just a few fixes.

**Comment by antoine-amara** [comment]
thanks for the review, I fix the typos.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

Feel free to open 1 big PR if you're translating lots of pages, with 1 commit per page  :-)

**Pull request merged**